### ISSUE TITLE
Accessibility / 508 Compliance: Missing Label

### ISSUE THREAD

**Issue by nxtwave** [HstarComponents/ngx-ckeditor#26]
The component works fine on my page for accessibility, but it fails accessibility tests from http://wave.webaim.org because the <textarea> tag in the template is missing a label so my page fails UAT.

Is it possible to add an aria-label attribute to the tag?

**Comment by hstarorg** [comment]
Good idea, I'll try to add it. Thanks.
